<ui version="4.0" >
 <class>UnittestStacktraceDialog</class>
 <widget class="QDialog" name="UnittestStacktraceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="testLabel" />
   </item>
   <item>
    <widget class="QTextEdit" name="traceback" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UnittestStacktraceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>33</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel" >
     <x>33</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UnittestStacktraceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>94</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel" >
     <x>95</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
